src/containers/Tenant/Diagnostics/TenantOverview/Healthcheck/HealthcheckPreview.tsx

```diff
@@ -45,17 +63,14 @@ export function HealthcheckPreview(props: HealthcheckPreviewProps) {
                         onClick={(event) => {
                             // FIXME: refactor card to remove the button from the anchor link.
                             event.preventDefault();
-                            onUpdate();
+                            refetch();
                         }}
-                        loading={loading}
+                        loading={isFetching}
                         view="flat-secondary"
                     >
                         <Icon data={ArrowsRotateRight} size={20} />
```

Comment:
Could you please remove refresh button on card? Discussed to abandon it in favor of refresh control in toolbar.